required/latex-lab/testfiles-math-luatex/mathml-luamml-1.tlg

```diff
@@ -0,0 +1,112 @@
+This is a generated file for the l3build validation system.
+Don't change this file in any respect.
+-------- mathml-luamml-1-luamml-mathml.html (start) ---------
+(mathml-luamml-1-luamml-mathml.html) <!DOCTYPE html>^^M
```

Comment:
all these `^^M` suggests (I think)  that on windows you are getting mixed format files with some unix and some dos line ends, which indicates an issue with the file writing somewhere which could probably be more cross platform

Reply:
the test passes also on git so it can't be windows dependant, can it?

Reply:
yes I know, it also passes for me using linux line ends, but that is because the lines in the file have unix line endings, just `#10` but the html part has been serialised with a `^^M` at the end of every line whereas the standard l3build  and tagpdf messages do not.

```
This is a generated file for the l3build validation system.
Don't change this file in any respect.
-------- mathml-luamml-1-luamml-mathml.html (start) ---------
(mathml-luamml-1-luamml-mathml.html) <!DOCTYPE html>^^M
<html>^^M
^^M
<div>^^M
<h2>\mml 1</h2>^^M
<p>\begin {equation}\sqrt {x^2}=\lvert x\rvert \end {equation}</p>^^M
<p>97E0ED65D7DB327BEE6496118E96C5B1</p>^^M
^^M
<math display="block" xmlns="http://www.w3.org/1998/Math/MathML">^^M
 <mo lspace="0" rspace="0"/>^^M
 <msqrt>^^M
 <msup>^^M
 <mi>^^M
....
</div>^^M
^^M
</html>^^M
-------- mathml-luamml-1-luamml-mathml.html (end) -----------
Inserting mathml with Hash 97E0ED65D7DB327BEE6496118E96C5B1
====>subpart splitting deactivated
====>grabbed math=macro:->\sqrt {x^2}=\lvert x\rvert 
Inserting mathml with Hash 626AD6FB6CF94CF1BE7FF5A7B8335B53
====>subpart splitting deactivated
====>grabbed math=macro:->\sqrt {x^2}=\lvert x\rvert 
Inserting mathml with Hash 59E43DDDF43026AE640CBD33D3EF4739
====>subpart splitting deactivated
====>grabbed math=macro:->\varGamma 
[1
] (mathml-luamml-1.aux)
MathML statistic
================
==> 3 MathML fragments read
==> 3 different MathML fragments
==> 3 math fragments found
==> 3 fitting MathML AF found
==> 3 MathML AF attached
Package tagpdf Info: Finalizing the tagging structure:
(tagpdf)             Writing out ~12 structure objects
(tagpdf)             with ~9 'MC' leaf nodes.
(tagpdf)             Be patient if there are lots of objects!
Package tagpdf Info: writing ParentTree
Package tagpdf Info: writing IDTree
Package tagpdf Info: writing RoleMap
Package tagpdf Info: writing ClassMap
Package tagpdf Info: writing NameSpaces
Package tagpdf Info: writing StructElems
Package tagpdf Info: writing Root

```

Reply:
What test, and how is passing decider for it?